engine/runtime/src/main/scala/org/enso/compiler/Passes.scala

```diff
@@ -49,6 +49,7 @@ class Passes(
       LambdaShorthandToLambda,
       ImportSymbolAnalysis,
       AmbiguousImportsAnalysis,
+      PrivateModuleAnalysis.MODULE$,
```

Comment:
You don't have to use `MODULE$` name as Scala does. The dollar at the end means a _synthetic field_, which is not the case. I'd call a `public static PrivateModuleAnalysis.getDefault()` method here myself.

Reply:
Good point. Refactored to classical singleton pattern with human readable method and field names in b15ddde05f6f15bb59d14f8cfed5bf17afdffbd0